hive/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -163,6 +169,111 @@ public void renameTable(TableIdentifier from, TableIdentifier to) {
     }
   }
 
+  @Override
+  public boolean createNamespace(Namespace namespace) {
+    Preconditions.checkArgument(!namespace.isEmpty(),
+        "Namespace could not empty. ");
```

Comment:
The recommendation about error messages should be applied here as well.

Reply:
thanks Ryan , now has fixed it.